scripts/verify-knife

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# Verify mmfile $1 (default "set.mm") and discouraged file $2 (default
+# "discouraged") using metamath-knife.
+# Usage:
+# verify-knife [source=set.mm] [discouraged_file=discouraged]
+
+source="${1:-set.mm}"
+discouraged_file="${2:-discouraged}"
+
+metamath-knife ${source} --verify \
+                         --verify-parse-stmt \
+                         --verify-usage \
+                         --verify-markup
+
+if [ $? != 0 ]; then
```

Comment:
It is slightly better to use -ne instead of !=, because != does a string comparison, which needs a prior conversion from integer to string, while -ne does a numeric comparison.

Reply:
Done.